tests/Microsoft.Bot.Builder.TestBot.WebApi/App_Start/BotConfig.cs

```diff
@@ -1,13 +1,20 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
+﻿// <copyright file="BotConfig.cs" company="PlaceholderCompany">
```

Comment:
This is not the correct copyright format. The former  (MIT License) is the correct one.

Reply:
Fixed